src/backend/common/account-access.js

```diff
@@ -215,16 +215,26 @@ frontendCommunicator.on("getAccounts", () => {
     return cache;
 });
 
-frontendCommunicator.on("logoutAccount", accountType => {
+frontendCommunicator.on("logoutAccount", (accountType) => {
     logger.debug("got logout request for", accountType);
     removeAccount(accountType);
 });
 
+function setStreamerTokenIssue() {
+    streamerTokenIssue = true;
+}
+
+function setBotTokenIssue() {
+    botTokenIssue = true;
+}
+
```

Comment:
To be consistent with other functions of this type, let's consolidate this down to a single function that takes an `accountType` parameter. Something like this should work:

```js
function setAccountTokenIssue(accountType) {
    if (accountType === 'streamer') {
        streamerTokenIssue = true;
    } else if (accounType === 'bot') {
        botTokenIssue = true;
    } else {
        throw new Error('invalid account type')
    }
}
```